cmd/screenshot/server.go

```diff
@@ -19,8 +22,9 @@ func newServer(conf repository.Config, handler http.Handler) *server {
 	return &server{
 		config: conf,
 		srv: &http.Server{
-			Handler: handler,
-			Addr:    conf.Addr(),
+			Handler:           handler,
+			Addr:              conf.Addr(),
+			ReadHeaderTimeout: readHeaderTimeout,
```

Comment:
fix

```
cmd/screenshot/server.go:21:9: G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server (gosec)
                srv: &http.Server{
                        Handler: handler,
                        Addr:    conf.Addr(),
                },
```